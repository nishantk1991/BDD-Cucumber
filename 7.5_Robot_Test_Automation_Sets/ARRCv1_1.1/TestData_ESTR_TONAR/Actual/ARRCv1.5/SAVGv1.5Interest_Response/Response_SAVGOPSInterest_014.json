{
  "comment": " The most recent published TONAR RFR rate is available as-of: 2020-05-22. Computed time : 2021-11-23T06:51:03+00:00. Computed time is the time stamp when the calculation was completed.",
  "lastUpdatedFromSource": "2021-10-29T00:00:00+00:00",
  "legalDisclaimer": "Use of TONAR is subject to the Bank of Japan's Terms of Use ; see https://www.boj.or.jp/en/statistics/outline/exp/data/exmutan2.pdf.",
  "interestCalculatorCriteria": {
    "initialPrincipalBalance": 1000000000,
    "interestPeriodStartDate": "2020-05-13",
    "interestPeriodEndDate": "2020-06-05",
    "riskFreeRate": "TONAR",
    "riskFreeRateFloor": 1,
    "lookBackDays": 4,
    "lockoutDays": 0,
    "observationPeriodShift": true,
    "spread": -1,
    "spreadAdjustment": -1,
    "dayCountConvention": "ACT/365",
    "rateRoundingDecimalPoints": 11,
    "legacyRiskFreeRateFloor": 1,
    "paymentRounding": "NEAREST",
    "principalAmountChanges": [
      {
        "date": "2020-05-13",
        "principalAmountChange": -10000000
      },
      {
        "date": "2020-05-22",
        "principalAmountChange": 100000
      },
      {
        "date": "2020-05-24",
        "principalAmountChange": 100000
      },
      {
        "date": "2020-05-30",
        "principalAmountChange": -200000
      }
    ],
    "showDailyDetails": true,
    "calculateTillDate": "2020-05-29"
  },
  "currencyCode": "JPY",
  "interestSummary": [
    {
      "interestPeriodDate": "2020-05-29",
      "interestPeriodDays": 19,
      "observationPeriodDate": "2020-05-22",
      "observationPeriodDays": 20,
      "principalBalance": 990200000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -461134.25
    },
    {
      "interestPeriodDate": "2020-05-28",
      "interestPeriodDays": 16,
      "observationPeriodDate": "2020-05-21",
      "observationPeriodDays": 16,
      "principalBalance": 990200000.0,
      "publishedRiskFreeRate": -0.0684,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -434005.48
    },
    {
      "interestPeriodDate": "2020-05-27",
      "interestPeriodDays": 15,
      "observationPeriodDate": "2020-05-20",
      "observationPeriodDays": 15,
      "principalBalance": 990200000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -406876.71
    },
    {
      "interestPeriodDate": "2020-05-26",
      "interestPeriodDays": 14,
      "observationPeriodDate": "2020-05-19",
      "observationPeriodDays": 14,
      "principalBalance": 990200000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -379747.95
    },
    {
      "interestPeriodDate": "2020-05-24",
      "interestPeriodDays": 13,
      "observationPeriodDate": "2020-05-18",
      "observationPeriodDays": 13,
      "principalBalance": 990200000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -352619.18
    },
    {
      "interestPeriodDate": "2020-05-22",
      "interestPeriodDays": 11,
      "observationPeriodDate": "2020-05-18",
      "observationPeriodDays": 13,
      "principalBalance": 990100000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -298361.64
    },
    {
      "interestPeriodDate": "2020-05-21",
      "interestPeriodDays": 9,
      "observationPeriodDate": "2020-05-15",
      "observationPeriodDays": 12,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -244109.59
    },
    {
      "interestPeriodDate": "2020-05-20",
      "interestPeriodDays": 8,
      "observationPeriodDate": "2020-05-14",
      "observationPeriodDays": 9,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": -0.0672,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -216986.3
    },
    {
      "interestPeriodDate": "2020-05-19",
      "interestPeriodDays": 7,
      "observationPeriodDate": "2020-05-13",
      "observationPeriodDays": 8,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": -0.0684,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -189863.01
    },
    {
      "interestPeriodDate": "2020-05-18",
      "interestPeriodDays": 6,
      "observationPeriodDate": "2020-05-12",
      "observationPeriodDays": 7,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": -0.232,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -162739.73
    },
    {
      "interestPeriodDate": "2020-05-15",
      "interestPeriodDays": 5,
      "observationPeriodDate": "2020-05-11",
      "observationPeriodDays": 6,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": 0.11,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -135616.44
    },
    {
      "interestPeriodDate": "2020-05-14",
      "interestPeriodDays": 2,
      "observationPeriodDate": "2020-05-07",
      "observationPeriodDays": 5,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": 0,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -54246.58
    },
    {
      "interestPeriodDate": "2020-05-13",
      "interestPeriodDays": 1,
      "observationPeriodDate": "2020-05-06",
      "observationPeriodDays": 1,
      "principalBalance": 990000000.0,
      "publishedRiskFreeRate": 0.0111,
      "rateApplied": 1,
      "riskFreeRateFloorApplied": true,
      "simpleAverageRate": 1.0,
      "allInRate": -1.0,
      "cummulativeInterestAmount": -27123.29
    }
  ]
}