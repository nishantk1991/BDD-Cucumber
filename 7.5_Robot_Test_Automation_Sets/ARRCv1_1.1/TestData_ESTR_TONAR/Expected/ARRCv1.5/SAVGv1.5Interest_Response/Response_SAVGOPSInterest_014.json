{
    "comment": " Computed time : 2021-03-17T10:34:45+00:00. Computed time is the time stamp when the calculation was completed.",
    "lastUpdatedFromSource": "2020-08-05T10:04:15+00:00",
    "legalDisclaimer": "The TONAR benchmark is used subject to the terms of the UK Open Government Licence v.3.0 (https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/) and the Bank of England’s disclaimer for TONAR (set out below)*. The “TONAR” mark is used with the consent of the Bank of England (the benchmark administrator of TONAR), and the use of such mark does not imply or express any approval or endorsement by the Bank of England. “Bank of England” and “TONAR” are registered trade marks of the Bank of England. The Bank of England has no liability for your use of the calculator service herein.\n*Bank of England’s disclaimer for TONAR: TONAR is copyright the Bank of England. The trade marks “Bank of England” and “TONAR” are registered trade marks of the Bank of England. All Rights Reserved. The TONAR benchmark includes the proprietary information of the Bank of England and the data comprising the benchmark may not be copied or distributed except as specifically authorised. The TONAR benchmark is not intended to be relied upon as authoritative or taken in substitution for the exercise of judgement. The TONAR benchmark is not, and should not be construed as, an offer, bid or solicitation in relation to any financial instrument. The Bank of England does not guarantee, and expressly disclaims any liability for, and makes no representations or warranties whether express or implied, as to the currency, accuracy, timeliness, completeness or fitness for any particular purpose of the TONAR benchmark. The Bank of England accepts no liability whatsoever for any loss (including, but not limited to any direct, indirect or consequential loss, whether or not such loss is foreseeable and whether or not the Bank of England has been appraised of the use to which the TONAR benchmark will be put) howsoever arising from the use, the timeliness of delivery or the failure of delivery of the TONAR benchmark.",
    "interestCalculatorCriteria": {
        "initialPrincipalBalance": 1000000000,
        "interestPeriodStartDate": "2020-05-13",
        "interestPeriodEndDate": "2020-06-05",
        "riskFreeRate": "TONAR",
        "riskFreeRateFloor": 1,
        "lookBackDays": 4,
        "lockoutDays": 0,
        "observationPeriodShift": true,
        "spread": -1,
        "spreadAdjustment": -1,
        "dayCountConvention": "ACT/365",
        "rateRoundingDecimalPoints": 11,
        "legacyRiskFreeRateFloor": 1,
        "paymentRounding": "NEAREST",
        "principalAmountChanges": [
            {
                "date": "2020-05-13",
                "principalAmountChange": -10000000
            },
            {
                "date": "2020-05-22",
                "principalAmountChange": 100000
            },
            {
                "date": "2020-05-24",
                "principalAmountChange": 100000
            },
            {
                "date": "2020-05-30",
                "principalAmountChange": -200000
            }
        ],
        "showDailyDetails": true,
        "calculateTillDate": "2020-05-29"
    },
    "currencyCode": "JPY",
    "interestSummary": [
        {
            "interestPeriodDate": "2020-05-29",
            "interestPeriodDays": 19,
            "observationPeriodDate": "2020-05-22",
            "observationPeriodDays": 20,
            "principalBalance": 990200000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -461134.25
        },
        {
            "interestPeriodDate": "2020-05-28",
            "interestPeriodDays": 16,
            "observationPeriodDate": "2020-05-21",
            "observationPeriodDays": 16,
            "principalBalance": 990200000.00,
            "publishedRiskFreeRate": -0.0684,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -434005.48
        },
        {
            "interestPeriodDate": "2020-05-27",
            "interestPeriodDays": 15,
            "observationPeriodDate": "2020-05-20",
            "observationPeriodDays": 15,
            "principalBalance": 990200000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -406876.71
        },
        {
            "interestPeriodDate": "2020-05-26",
            "interestPeriodDays": 14,
            "observationPeriodDate": "2020-05-19",
            "observationPeriodDays": 14,
            "principalBalance": 990200000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -379747.95
        },
        {
            "interestPeriodDate": "2020-05-24",
            "interestPeriodDays": 13,
            "observationPeriodDate": "2020-05-18",
            "observationPeriodDays": 13,
            "principalBalance": 990200000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -352619.18
        },
        {
            "interestPeriodDate": "2020-05-22",
            "interestPeriodDays": 11,
            "observationPeriodDate": "2020-05-18",
            "observationPeriodDays": 13,
            "principalBalance": 990100000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -298361.64
        },
        {
            "interestPeriodDate": "2020-05-21",
            "interestPeriodDays": 9,
            "observationPeriodDate": "2020-05-15",
            "observationPeriodDays": 12,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -244109.59
        },
        {
            "interestPeriodDate": "2020-05-20",
            "interestPeriodDays": 8,
            "observationPeriodDate": "2020-05-14",
            "observationPeriodDays": 9,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": -0.0672,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -216986.30
        },
        {
            "interestPeriodDate": "2020-05-19",
            "interestPeriodDays": 7,
            "observationPeriodDate": "2020-05-13",
            "observationPeriodDays": 8,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": -0.0684,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -189863.01
        },
        {
            "interestPeriodDate": "2020-05-18",
            "interestPeriodDays": 6,
            "observationPeriodDate": "2020-05-12",
            "observationPeriodDays": 7,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": -0.232,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -162739.73
        },
        {
            "interestPeriodDate": "2020-05-15",
            "interestPeriodDays": 5,
            "observationPeriodDate": "2020-05-11",
            "observationPeriodDays": 6,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": 0.11,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -135616.44
        },
        {
            "interestPeriodDate": "2020-05-14",
            "interestPeriodDays": 2,
            "observationPeriodDate": "2020-05-07",
            "observationPeriodDays": 5,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": 0,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -54246.58
        },
        {
            "interestPeriodDate": "2020-05-13",
            "interestPeriodDays": 1,
            "observationPeriodDate": "2020-05-06",
            "observationPeriodDays": 1,
            "principalBalance": 990000000.00,
            "publishedRiskFreeRate": 0.0111,
            "rateApplied": 1,
            "riskFreeRateFloorApplied": true,
            "simpleAverageRate": 1.00000000000,
            "allInRate": -1.00000000000,
            "cummulativeInterestAmount": -27123.29
        }
    ]
}